control : { 
            'feedback_gain'    : [ 36.38497699, 0, 0, 48.31898852, 0, 0, 
                                    0, 36.36882206, 0, 0, 48.31492434, 0,
                                    0, 0, 1.68748, 0, 0, 2.17242],

            'feedforward_gain' : [ 4.79184, 0.0, 0.0, 
                                   0.0, 4.78888,   0.0,
                                   0.0, 0.0, 0.2222],

            'output_matrix'    : [ 1, 0, 0, 0, 0, 0, 
                                   0, 1, 0, 0, 0, 0,
                                   0, 0, 1, 0, 0, 0]
          }


command_governor : { 
                      'k0'    : 100, 
                      'Psi'   : [2, 0.0, 0.0, 0.0, 1.0, 0.0, 0.0, 0.0, 10.0],
                      'Tc_cg' : 0.1
                   }
                   
system : {
            'Phi'       :          [0.984247484648385, 0, 0, 0.0807861213625316, 0, 0, 0.00206519097670078, 0, 0,
                                    0, 0.984254273943164, 0, 0, 0.0807878511891364, 0, 0, 0.00206394560433122, 0, 
                                    0, 0, 0.9874, 0, 0, 0.0807876004095419, 0, 0, 0.00206364788788813,
                                    -0.293867257861998, 0, 0, 0.636818670849580, 0, 0, 0.0384299099910308, 0, 0,
                                    0, -0.293742697769447, 0, 0, 0.636850615731820, 0, 0, 0.0384070276793408, 0,
                                    0, 0, -0.293721267629194, 0, 0, 0.636846691660727, 0, 0, 0.0384014509415136,
                                    -0.0994566530257753, 0, 0, -0.00434137808075775, 0, 0, 0.999928681811085, 0, 0,
                                    0, -0.0994568890350298, 0, 0, -0.00434143802372067, 0, 0, 0.999928725073154, 0,
                                    0, 0, -0.0994569277301780, 0, 0, -0.00434142869254123, 0, 0, 0.999928735319247],
             
             'G'        :          [7.13181889151415e-05, 0, 0,
                                    0, 7.12749268460947e-05, 0,
                                    0, 0, 7.12646807527457e-05,
                                    0.00206519097670078, 0, 0,
                                    0, 0.00206394560433122, 0,
                                    0, 0, 0.00206364788788813,
                                    0.0999981790126124, 0, 0,
                                    0, 0.0999981801210440, 0,
                                    0, 0, 0.0999981803821057],
            }
